MAINTAINERS.yml

```diff
@@ -1569,21 +1569,26 @@ Release Notes:
   collaborators:
     - tomi-font
     - fabiobaltieri
+    - ubieda
   files:
     - doc/hardware/peripherals/gnss.rst
     - drivers/gnss/
     - include/zephyr/data/navigation.h
     - include/zephyr/drivers/gnss.h
     - include/zephyr/drivers/gnss/
+    - include/zephyr/gnss/
     - dts/bindings/gnss/
     - include/zephyr/dt-bindings/gnss/
     - tests/drivers/build_all/gnss/
     - tests/drivers/gnss/
+    - tests/gnss/
```

Comment:
```suggestion
    - tests/subsys/gnss/
```

Reply:
Yep I realize I missed the folder. Pushed now fix.